examples/aapp-processing.yaml-template

```diff
@@ -83,13 +83,12 @@ aapp_static_configuration:
     'amsu-b': 'amsub'
     'hirs/4': 'hirs'
     'hirs/3': 'hirs'
-    'mhs': 'mhs'
```

Comment:
Would it break functionality to leave it there? I think it is there for completeness.

Reply:
Look up a couple of lines — it's there twice in the same dictionary! That can't add any completeness?

Reply:
ok, sorry, I was reviewing on the phone, which doesn't let me look above and below. Sounds good then.